packages/muelu/test/scaling/CMakeLists.txt

```diff
@@ -288,6 +288,14 @@ IF (${PACKAGE_NAME}_HAVE_TPETRA_SOLVER_STACK)
     NUM_MPI_PROCS 4
     COMM mpi # HAVE_MPI required
   )
+  TRIBITS_ADD_TEST(
+    Driver
+    NAME "GeneralBlockSmoothing"
```

Comment:
Won't tribits complain if this test has the same name as the test above it?

Reply:
Not sure ... but it is probably better to change the name which I just did by prepending the word Serial to the name.